app/editor/src/store/hooks/signalr/useApiHub.ts

```diff
@@ -34,12 +38,11 @@ export interface IHubController {
  * @returns Hook controller.
  */
 export const useApiHub = (): IHubController => {
-  const [state, setState] = React.useState(hub?.state ?? HubConnectionState.Disconnected);
-  const [, app] = useAppStore();
+  const [{ hubState }, { addError, changeHubState }] = useAppStore();
```

Comment:
We need to use redux store for maintaining the hub state, otherwise it won't update.